pydantic/networks.py

```diff
@@ -153,6 +154,7 @@ def __init__(
         path: Optional[str] = None,
         query: Optional[str] = None,
         fragment: Optional[str] = None,
+        plus: bool = False,
```

Comment:
```suggestion
        quote_plus: bool = False,
```
might be a clearer name.

Reply:
Also, is there a good reason not to have `quote_plus = True` as the default?

Reply:
Please add `quote_plus` to `stricturl`

Reply:
> Also, is there a good reason not to have `quote_plus = True` as the default?

Just conforming to the standard ([RFC 3986](https://www.ietf.org/rfc/rfc3986.txt))